include/ck/wrapper/utils/layout_utils.hpp

```diff
@@ -106,9 +118,13 @@ __host__ __device__ constexpr auto make_layout(const Shape& shape)
 
 // Layout helpers
 // get
-// Get dim (could be returned from get with empty Idxs)
+
 /**
  * \private
+ * \brief Get dim (could be returned from get with empty Idxs).
```

Comment:
what does it mean "could be returned from get with empty Idxs"? Why Idxs is capitalized? Maybe you could describe it so it's not confusing?